operator/src/test/java/org/keycloak/operator/utils/K8sUtils.java

```diff
@@ -67,4 +73,39 @@ public static void waitForKeycloakToBeReady(KubernetesClient client, Keycloak kc
                     CRAssert.assertKeycloakStatusCondition(currentKc, KeycloakStatusCondition.HAS_ERRORS, false);
                 });
     }
+
+    public static String inClusterCurl(KubernetesClient k8sclient, String namespace, String url) {
```

Comment:
I like it that it's generic now. 👍

Reply:
and is going to be much more generic:
https://github.com/java-operator-sdk/java-operator-sdk/pull/966